docs/source/10min_cudf_cupy.ipynb

```diff
@@ -0,0 +1,504 @@
+    "    arr = cp.random.normal(5,5, nelem)\n",
+    "    arr[cp.random.choice(arr.shape[0], nelem-nonzero, replace=False)] = 0\n",
+    "    df['a'+str(i)] = arr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    a0                 a1                  a2                 a3                 a4                 a5   a6 ...                a19\n",
+      "0  0.0                0.0                 0.0                0.0                0.0                0.0  0.0 ...                0.0\n",
+      "1  0.0                0.0                 0.0                0.0                0.0    9.6816655366294  0.0 ...  10.17953305675039\n",
+      "2  0.0  7.544019774751115                 0.0                0.0                0.0                0.0  0.0 ...                0.0\n",
+      "3  0.0                0.0  0.2411281512639345  20.44337627253567  7.129263932399107                0.0  0.0 ...                0.0\n",
+      "4  0.0                0.0   6.671876913849459                0.0                0.0  2.569008638616691  0.0 ...                0.0\n",
+      "[12 more columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.head())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/conda/envs/rapids/lib/python3.6/site-packages/cudf/io/dlpack.py:83: UserWarning: WARNING: cuDF to_dlpack() produces column-major (Fortran order) output. If the output tensor needs to be row major, transpose the output of this function.\n",
+      "  return cpp_dlpack.to_dlpack(gdf_cols)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 204 ms, sys: 76 ms, total: 280 ms\n",
+      "Wall time: 328 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "%time sparse_data = cudf_to_cupy_sparse_matrix(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From here, we could continue our workflow with a CuPy sparse matrix."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Interweaving CuDF and CuPy for Smooth PyData Workflows\n",
+    "\n",
+    "RAPIDS libraries and the entire GPU PyData ecosystem are developing quickly, but sometimes a one library may not have the functionality you need. One example of this might be taking the row-wise sum (or mean) of a Pandas DataFrame. cuDF's support for row-wise operations isn't mature, so you'd need to either transpose the DataFrame or write a UDF and explicitly calculate the sum across each row. Transpoing could lead to hundreds of thousands of columns depending on your data size, which cuDF isn't optimized for currently, and writing a UDF can be a bit more time intensive.\n",
```

Comment:
> Transpoing could lead to hundreds of thousands of columns depending on your data size, which cuDF isn't optimized for currently

```suggestion
    "RAPIDS libraries and the entire GPU PyData ecosystem are developing quickly, but sometimes a one library may not have the functionality you need. One example of this might be taking the row-wise sum (or mean) of a Pandas DataFrame. cuDF's support for row-wise operations isn't mature, so you'd need to either transpose the DataFrame or write a UDF and explicitly calculate the sum across each row. Transposing could lead to hundreds of thousands of columns (which cuDF wouldn't perform well with) depending on your data's shape, and writing a UDF can be time intensive.\n",
```